.github/workflows/website-validation.yml

```diff
@@ -22,8 +22,6 @@ jobs:
         with:
           submodules: false
       - uses: pnpm/action-setup@d882d12c64e032187b2edb46d3a0d003b7a43598 # ratchet:pnpm/action-setup@v2
-        with:
-          version: "^7"
```

Comment:
This is required if the package.json does not specify a version , and optional if it does. Since a version is now provided in the package.json, this can be removed, see https://github.com/pnpm/action-setup?tab=readme-ov-file#version for details.